experiments/AMIP/moist_mpi_earth/Project.toml

```diff
@@ -42,7 +42,6 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
 [compat]
 ClimaCore = "0.10.3"
-RRTMGP = "0.3.2"
```

Comment:
We actually need this unless we upgrade to the new clima parameter handling, but I think that may be better as a separate PR